.github/workflows/stackhpc-all-in-one.yml

```diff
@@ -396,13 +396,15 @@ jobs:
           KAYOBE_AUTOMATION_SSH_PRIVATE_KEY: ${{ steps.ssh_key.outputs.ssh_key }}
         if: inputs.upgrade
 
+      # NOTE(Alex-Welsh): configure host with selinux_state=disabled, because
+      # Antelope images are not built as permissive
```

Comment:
Seems fine for now, can always remove this once we fix the AIOs failing to reboot.